auth-web/src/components/auth/account-settings/statement/Statements.vue

```diff
@@ -96,171 +114,246 @@
 </template>
 
 <script lang="ts">
-import { Account, Pages } from '@/util/constants'
-import { Component, Mixins, Prop, Watch } from 'vue-property-decorator'
+import { Account, LDFlags, Pages } from '@/util/constants'
+import { ComputedRef, PropType, Ref, computed, defineComponent, onMounted, ref, watch } from '@vue/composition-api'
 import { Member, MembershipType, Organization } from '@/models/Organization'
-import { StatementFilterParams, StatementListItem, StatementListResponse } from '@/models/statement'
-import { mapActions, mapState } from 'pinia'
+import { StatementFilterParams, StatementListItem } from '@/models/statement'
 import AccountChangeMixin from '@/components/auth/mixins/AccountChangeMixin.vue'
 import CommonUtils from '@/util/common-util'
+import DocumentService from '@/services/document.services'
+import LaunchDarklyService from 'sbc-common-components/src/services/launchdarkly.services'
+import ModalDialog from '@/components/auth/common/ModalDialog.vue'
 import StatementsSettings from '@/components/auth/account-settings/statement/StatementsSettings.vue'
 import moment from 'moment'
+import { useAccountChangeHandler } from '@/composables'
 import { useOrgStore } from '@/stores/org'
 
-@Component({
-  components: {
-    StatementsSettings
-  },
-  methods: {
-    ...mapActions(useOrgStore, [
-      'getStatementsList',
-      'getStatement'
-    ])
+export default defineComponent({
+  name: 'StatementsView',
+  components: { StatementsSettings },
+  mixins: [AccountChangeMixin],
+  props: {
+    orgId: {
+      type: String as PropType<string>,
+      default: ''
+    }
   },
-  computed: {
-    ...mapState(useOrgStore, [
-      'currentOrganization',
-      'currentMembership'
+  setup (props, { root }) {
+    const orgStore = useOrgStore()
+    const currentOrganization: ComputedRef<Organization> = computed(() => orgStore.currentOrganization)
+    const currentMembership: ComputedRef<Member> = computed(() => orgStore.currentMembership)
```

Comment:
suggestion: use reactive, use toRefs after, you can even nest computeds in there as well gets rid of `.value`